### PR TITLE
Update Releasing.md

### PR DESCRIPTION
**Pull request by ceyhun**
Added few more items to the checklist.

### CODE REVIEW

File: Releasing.md [hunk 1]
```diff
@@ -48,6 +48,18 @@ cut a new release.
 <p>o Run the release script in release-toolkit-gutenberg-mobile: <code>./release_automation.sh</code>. This will take care of creating the gutenberg and gutenberg-mobile release PRs as well as WPAndroid and WPiOS integration PRs.</p>
 <!-- /wp:paragraph -->
 
+<!-- wp:paragraph -->
+<p>o Trigger an installable build on WPiOS PR.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Fill in the missing parts in gutenberg-mobile PR description.</p>
```
Comment (jd-alexander) [inline Releasing.md]: Nitpick :) 
```suggestion
<p>o Fill in the missing parts of the gutenberg-mobile PR description.</p>
```

File: Releasing.md [hunk 2]
```diff
@@ -48,6 +48,18 @@ cut a new release.
...
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Mark all 4 PRs ready for review, requesting reviews from your buddy.</p>
```
Comment (jd-alexander) [inline Releasing.md]: I modified this sentence a bit. Let me know what you think.
```suggestion
<p>o Mark all 4 PRs ready for review and request reviews for them from your release wrangler buddy.</p>
```

Comment (ceyhun) [reply Releasing.md]: Sounds good! Thanks 👍

**Review by guarani — APPROVED** [review]
Thanks @ceyhun!

**Pull request merged**